cmd/expired-authz-purger/main.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2014 ISRG.  All rights reserved
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/cactus/go-statsd-client/statsd"
+	"github.com/codegangsta/cli"
```

Comment:
I'd like to migrate away from `codegangsta/cli` since we don't need it for much, and it's one more dependency. So I'd recommend skipping it for new binaries.